src/datepicker/docs/readme.md

```diff
@@ -144,7 +144,11 @@ Options for the uib-datepicker must be passed as JSON using the `datepicker-opti
   _(Default: `yyyy-MM-dd`)_ -
   The format for displayed dates. This string can take string literals by surrounding the value with single quotes, i.e. `yyyy-MM-dd h 'o\'clock'`
 
-### Keyboard support ###
+ * `alt-input-formats`
+  _(Default: [])_:
+  A list of alternate formats acceptable for manual entry.
+
+### Keyboard Support ###s
```

Comment:
remove that extra `s` and lowercase that `support` (not sure why you even touched that anyway).

Also the `[]` on the default needs to be between backsticks.